### ISSUE TITLE
预测速度的提升

### ISSUE THREAD

**Issue by skywindy** [Determined22/zh-NER-TF#79]
自己训练了一个模型，效果挺好但是预测比较慢，请问这种可以怎么解决

**Comment by skywindy** [comment]
大神你好，如果在预测阶段，我把模型中的batch_size值调大一些，通过调用 predict_one_batch 函数进行批量预测，会不会预测的更快一些，也不影响模型的准确率效果呢？

def predict_one_batch(self, sess, seqs):
        """
        :param sess:
        :param seqs:
        :return: label_list
                 seq_len_list
        """
        feed_dict, seq_len_list = self.get_feed_dict(seqs, dropout=1.0)

        if self.CRF:
            logits, transition_params = sess.run([self.logits, self.transition_params],
                                                 feed_dict=feed_dict)
            label_list = []
            for logit, seq_len in zip(logits, seq_len_list):
                viterbi_seq, _ = viterbi_decode(logit[:seq_len], transition_params)
                label_list.append(viterbi_seq)
            return label_list, seq_len_list

        else:
            label_list = sess.run(self.labels_softmax_, feed_dict=feed_dict)
            return label_list, seq_len_list
